lib/AppInfo/Application.php

```diff
@@ -28,14 +28,30 @@ class Application extends App implements IBootstrap {
 
 	public const APP_ID = 'context_chat';
 
-	public const CWYD_DEFAULT_REQUEST_TIMEOUT = 60 * 120;
+	public const CC_DEFAULT_REQUEST_TIMEOUT = 60 * 120;
 	// max size per file + max size of the batch of files to be embedded in a single request
-	public const CWYD_MAX_SIZE = 20 * 1024 * 1024; // 20MB
-	public const CWYD_MAX_FILES = 100;
+	public const CC_MAX_SIZE = 20 * 1024 * 1024; // 20MB
+	public const CC_MAX_FILES = 100;
 	public const MIMETYPES = [
 		'text/plain',
 		'text/markdown',
 		'application/json',
+		'application/pdf',
+		'text/csv',
+		'application/epub+zip',
+		'application/vnd.openxmlformats-officedocument.wordprocessingml.document',
+		'application/vnd.ms-powerpoint',
+		'application/vnd.openxmlformats-officedocument.presentationml.presentation',
+		'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet',
+		'application/vnd.oasis.opendocument.spreadsheet',
+		'application/vnd.ms-excel.sheet.macroEnabled.12',
+		'application/vnd.oasis.opendocument.text',
+		'text/rtf',
+		'text/x-rst',
+		'application/xml',
+		'message/rfc822',
+		'application/vnd.ms-outlook',
+		'text/org',
```

Comment:
Woop, woop, for more supported file formats! Thank you for the work on that!

Reply:
The pleasure is all mine :)